michael/src/App.js

```diff
@@ -1,6 +1,16 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import gifList from './gifList';
+
+class App extends Component {
```

Comment:
You'll want to delete this - Previously, React Components were class based (like this), now React Components are shifting to function based components (like on line 15). Right now React may be confused if you're calling <App/> since it doesn't know if you're calling App (the class on line 5) or App (the function on line 15)

Move all of your code from line 7 - 11 to where line 16 currently is, and delete lines 5 - 13